### DISCUSSION TITLE
On windows 10 path separators (slashes)  are mixed up?

### DISCUSSION THREAD

**Discussion by Kristijan10048** [isl-org/Open3D#3833] [Q&A]
Hi all,

I am quite new to this project. I was able to compile the entire solution under windows however i got some issues, not sure if this is a bug (although looks like it).

C:\Repository\Open3D\build\bin\Open3D\Release>Open3DViewer.exe
[Open3D Error] (void __cdecl open3d::visualization::rendering::EngineInstance::SetResourcePath(const class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &)) C:\Repository\Open3D\cpp\open3d\visualization\rendering\filament\FilamentEngine.cpp:66: Can't find resource directory: **C:\Repository\Open3D\build\bin\Open3D\Release/Open3DViewer.exe/../resources**

One level up is the resources folder. I used CMake for windows and Visual Studio 2019.

Kind regards

**Comment by StefanCognite** [comment]
Hi
I am also new to this project. I also got your problem.

I just fixed it by changing the function :

std::string FindResourcePath(int argc, const char* argv[]) in Application.cpp

At the first line in the function, I added 

return "C:/Repository/Open3D/build/bin/resources";

**Comment by germanros1987** [comment]
This should be solved on 0.17.0. Please, let us know if that's not working for you yet.
